saleor/graphql/notifications/mutations/external_notification_trigger.py

```diff
@@ -0,0 +1,142 @@
+import json
+
+import graphene
+
+from saleor.account.models import User
+from saleor.checkout.models import Checkout
+from saleor.core.exceptions import PermissionDenied
+from saleor.core.permissions import (
+    CheckoutPermissions,
+    OrderPermissions,
+    ProductPermissions,
+)
+from saleor.graphql.core.types.common import ExternalNotificationError
+from saleor.invoice.models import Invoice
+from saleor.order.models import Fulfillment, Order
+from saleor.product.models import Product, ProductVariant
+
+from ....core.permissions import AccountPermissions
+from ....graphql.utils import resolve_global_ids_to_primary_keys
+from ....webhook.payloads import (
+    generate_checkout_payload,
+    generate_customer_payload,
+    generate_fulfillment_lines_payload,
+    generate_invoice_payload,
+    generate_order_payload,
+    generate_product_payload,
+    generate_product_variant_payload,
+)
+from ...core.mutations import BaseMutation
+from ..inputs import ExternalNotificationTriggerInput
+
+
+class ExternalNotificationTrigger(BaseMutation):
+    PK__IN = "pk__in"
+    PAYLOAD_MAPPING = {
+        "Fulfillment": (
+            Fulfillment,
+            generate_fulfillment_lines_payload,
+            None,
+            PK__IN,
+            OrderPermissions.MANAGE_ORDERS,
+        ),
+        "ProductVariant": (
+            ProductVariant,
+            generate_product_variant_payload,
+            list,
+            PK__IN,
+            ProductPermissions.MANAGE_PRODUCTS,
+        ),
+        "Product": (
+            Product,
+            generate_product_payload,
+            None,
+            PK__IN,
+            ProductPermissions.MANAGE_PRODUCTS,
+        ),
+        "User": (
+            User,
+            generate_customer_payload,
+            None,
+            PK__IN,
+            AccountPermissions.MANAGE_USERS,
+        ),
+        "Checkout": (
+            Checkout,
+            generate_checkout_payload,
+            None,
+            "token__in",
+            CheckoutPermissions.MANAGE_CHECKOUTS,
+        ),
+        "Invoice": (
+            Invoice,
+            generate_invoice_payload,
+            None,
+            PK__IN,
+            OrderPermissions.MANAGE_ORDERS,
+        ),
+        "Order": (
+            Order,
+            generate_order_payload,
+            None,
+            PK__IN,
+            OrderPermissions.MANAGE_ORDERS,
+        ),
+    }
+
+    class Arguments:
+        input = ExternalNotificationTriggerInput(
+            required=True, description="Input for External Notification Trigger."
+        )
+        pluginId = graphene.String(description="The ID of notification plugin.")
+
+    class Meta:
+        description = "Send external notification to given customers."
```

Comment:
1. It's not only for customers.
2. Can we actually describe what the mutation does? I mean sth like: "Trigger sending a notification with the `notify` plugin method. Serializes nodes provided as `ids` parameter and includes this data in the notification payload."